idaes/core/util/config.py

```diff
@@ -107,6 +107,33 @@ def list_of_floats(arg):
     return lst
 
 
+def list_of_ints(arg):
+    '''Domain validator for lists of integers
```

Comment:
note that 'black' may correct these to triple-double quotes

Reply:
At some point ,we need to just bite the bullet and run `black` over the code based and then add a test to enforce that standard from now on,

As it is, I am hesitant to run `black` over code in an actual development PR as it will obscure the real changes.